vc/vcutils.h

```diff
@@ -5,6 +5,7 @@
 
 #define MASK_32_BITS 4294967295
 
+void strip_endl(unsigned char *str_to_strip, size_t *len_str);
```

Comment:
If vcutils.c: strip_endl definition is changed, the prototype here should be changed too.